JenkinsFile_build_container.groovy

```diff
@@ -17,9 +17,8 @@
  * [1] https://www.gnu.org/software/classpath/license.html
  * [2] https://openjdk.org/legal/assembly-exception.html
  *
- * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
- * Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
- *******************************************************************************/
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+  *******************************************************************************/
```

Comment:
Please undo the change to line 21. Similar, unwanted changes were made to `JenkinsFile_build_doc.groovy` and `jenkinsFile_cleanup_staging.groovy`.

Reply:
Updated.